src/Interactive/Host/Interactive/Core/InteractiveHost.cs

```diff
@@ -175,36 +152,23 @@ private RemoteService TryStartProcess(string hostPath, CultureInfo culture, Canc
                     newProcessId = 0;
                 }
 
-                // sync:
-                while (!semaphore.WaitOne(_millisecondsTimeout))
-                {
-                    if (!CheckAlive(newProcess, hostPath))
-                    {
-                        return null;
-                    }
-
-                    WriteOutputInBackground(isError: false, string.Format(InteractiveHostResources.Attempt_to_connect_to_process_Sharp_0_failed_retrying, newProcessId));
-                    cancellationToken.ThrowIfCancellationRequested();
-                }
+                // (miziga) initializing pipe for replacement of remote channel
+                NamedPipeClientStream _clientStream = new NamedPipeClientStream(".", pipeName, PipeDirection.InOut, PipeOptions.Asynchronous);
```

Comment:
>_clientStream [](start = 38, length = 13)

Style: local variables should not start with `_`. 
Also you can use `var` here instead of `NamedPipeClientStream` as the type is clear from the right hand side.